packages/coinstac-api-server/src/auth-helpers.js

```diff
@@ -205,6 +245,64 @@ const helperFunctions = {
       res(Boom.unauthorized('Incorrect username or password.'));
     }
   },
+  /**
+   * Confirms that submitted email is valid
+   * @param {object} req request
+   * @param {object} res response
+   * @return {object} The requested object
+   */
+  async validateEmail(req, res) {
+    const db = database.getDbInstance();
+    const user = await db.collection('users').findOne({ email: req.payload.email });
+
+    if (!user) {
+      return res(Boom.badRequest('Invalid email'));
+    }
+
+    res(req.payload);
+  },
+  /**
+   * Confirms that submitted token is valid
+   * @param {object} req request
+   * @param {object} res response
+   * @return {object} The requested object
+   */
+  async validateResetToken(req, res) {
+    const db = database.getDbInstance();
+    const user = await db.collection('users').findOne({ passwordResetToken: req.payload.token });
+
+    if (!user) {
+      return res(Boom.badRequest('Invalid token'));
+    }
+
+    try {
+      const { email } = jwt.verify(req.payload.token, dbmap.cstacJWTSecret);
+      return helperFunctions.validateEmail({ payload: { email } }, res);
+    } catch (err) {
+      res(Boom.badRequest(err));
+    }
+  },
+  /**
+   * Reset password
+   * @param {object} password token for resetting password
+   * @param {object} password new password
+   * @return {object}
+   */
+  async resetPassword(token, password) {
+    const db = database.getDbInstance();
+
+    const newPassword = await helperFunctions.hashPassword(password);
+    await db.collection('users').findOneAndUpdate({
```

Comment:
Same thing here. You're not expecting the return from this function, so use `updateOne` instead.